### PR TITLE
Here are the docs for the portal, please review

### PR DESCRIPTION
**Pull request by mdeneva**
Initial content, might need to add more stuff

### CODE REVIEW

File: docs/concepts/portal/applications.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+---
+
+The __Applications__ tab displays all of your applications including the undeployed. For a detailed explanation of the view, refer to the table below:
+
+
+
+| Column        |      Description                               |  Possible Values                                                    |
+|:--------------|:-----------------------------------------------|:--------------------------------------------------------------------|
+| __App Name__  | The name of the app                            | `String` (can contain special characters)                           |
+| __Kind__      | Indicates .....                                | __Replica__ <br />  __StatefulSet__ <br /> __Deployment__           |
```
Comment (cvlc) [inline docs/concepts/portal/applications.md:13]: Indicates the `kind` of Kubernetes resource used to run your application

File: docs/concepts/portal/dashboard.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
+---
+title: " Dashboard Page Reference"
```
Comment (cvlc) [inline docs/concepts/portal/dashboard.md:2]: Extra space?

File: docs/concepts/portal/dashboard.md [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+linkTitle: "Dashboard Page Reference"
+---
+
+
+The Dashboard gives you an unified and summarised view of the application you have deployed. Top view all of your application scroll the Dashboard page. If there are any persistent volumes in error this will be indicated on the dashboard as a red side line next to the application name and will also give you the number of affected volumes 
+
+Your deployed application can be one of the following types:
+* Replica - <add_description_here>
```
Comment (cvlc) [inline docs/concepts/portal/dashboard.md:10]: A [ReplicaSet](https://kubernetes.io/docs/concepts/workloads/controllers/replicaset/) ensures that one or more pods are running at any given time, according to configuration. Usually, `ReplicaSet`s are managed by `Deployment`s.

File: docs/concepts/portal/dashboard.md [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+* StatefulSet - <add_description_here>
```
Comment (cvlc) [inline docs/concepts/portal/dashboard.md:11]: A [StatefulSet](https://kubernetes.io/docs/concepts/workloads/controllers/statefulset/) represents a stateful application that both manages one or more pods, ensures that they are running at any given time and provides certain guarantees about the order and uniqueness of the pods.

File: docs/concepts/portal/dashboard.md [hunk 5]
```diff
@@ -0,0 +1,14 @@
...
+* Deployment - <add_description_here>
```
Comment (cvlc) [inline docs/concepts/portal/dashboard.md:12]: A [Deployment](https://kubernetes.io/docs/concepts/workloads/controllers/deployment/) provides a desired state for one or more sets of pods without any particular guarantees about order or uniqueness.

File: docs/concepts/portal/dashboard.md (hunk 3 above)
Comment (mdeneva) [reply docs/concepts/portal/dashboard.md:10]: Thanks, will include

File: docs/concepts/portal/dashboard.md (hunk 4 above)
Comment (mdeneva) [reply docs/concepts/portal/dashboard.md:11]: Thanks,, useful description - will include

File: docs/concepts/portal/dashboard.md (hunk 5 above)
Comment (mdeneva) [reply docs/concepts/portal/dashboard.md:12]: Same

File: docs/concepts/portal/applications.md [hunk 6]
```diff
@@ -0,0 +1,23 @@
+---
+title: " Applications Page Reference"
+linkTitle: "Applications Page Reference"
+---
+
+The __Applications__ tab displays all of your applications including the undeployed. For a detailed explanation of the view, refer to the table below:
```
Comment (mvisintin) [inline docs/concepts/portal/applications.md:6]: `including the undeployed`

Not exactly sure what you're referring to here. The applications tab shows an aggregation of information pertaining all statefulsets deployed across all the connected clusters - so I am a bit unclear to what this statement pertaining deployments refers to.

Maybe deployed as exposed?

File: docs/concepts/portal/applications.md [hunk 7]
```diff
@@ -0,0 +1,23 @@
...
+
+
+
+| Column        |      Description                               |  Possible Values                                                    |
+|:--------------|:-----------------------------------------------|:--------------------------------------------------------------------|
+| __App Name__  | The name of the app                            | `String` (can contain special characters)                           |
+| __Kind__      | Indicates .....                                | __Replica__ <br />  __StatefulSet__ <br /> __Deployment__           |
+| __Pods__      | The number of pods for your application        | `Integer`                                                           |
+| __Pod Status__| Indicates the number of pods that are ready/syncing or with unknown/failed status| __Ready__ <br /> __Syncing__ <br /> __Unknown__ <br /> __Failed__   |
+| __PV Amount__ | Indicates the amount of PVs taken up by the app| `Integer`                                                           |
```
Comment (mvisintin) [inline docs/concepts/portal/applications.md:16]: `amount of PVs taken up by the app`

Think this should be _connected to the app_

File: docs/concepts/portal/dashboard.md [hunk 8]
```diff
@@ -0,0 +1,14 @@
+---
+title: " Dashboard Page Reference"
+linkTitle: "Dashboard Page Reference"
+---
+
+
+The Dashboard gives you an unified and summarised view of the application you have deployed. Top view all of your application scroll the Dashboard page. If there are any persistent volumes in error this will be indicated on the dashboard as a red side line next to the application name and will also give you the number of affected volumes
```
Comment (mvisintin) [inline docs/concepts/portal/dashboard.md:7]: `Top view all of your application scroll the Dashboard page.`

Don't think this sentence is needed. Also little typo.

File: docs/concepts/portal/installation-portal.md [hunk 9]
```diff
@@ -0,0 +1,50 @@
+---
+title: "Ondat Portal Installation Guide"
+linkTitle: "Ondat Portal Installation Guide"
+---
+
+
+# Setting Up a Local Cluster
```
Comment (mvisintin) [inline docs/concepts/portal/installation-portal.md:7]: This should be marked as an optional step.

File: docs/concepts/portal/installation-portal.md [hunk 10]
```diff
@@ -0,0 +1,50 @@
...
+---
+
+
+# Setting Up a Local Cluster 
+
+The following guide will take you through the process of setting up a cluster on your machine. If you are using an Apple M1 machine you need to run kind inside an amd64 VM hosted on a Cloud Service of your choice. Local emulation will not work on a machine using Apple M1. 
+
+## Prerequisites
+
+* You have set up your cluster using the following config file <cloud_init.yml>
```
Comment (mvisintin) [inline docs/concepts/portal/installation-portal.md:13]: Is the file or its contents included somewhere linkable in the documentation?

File: docs/concepts/portal/installation-portal.md [hunk 11]
```diff
@@ -0,0 +1,50 @@
...
+`multipass launch --name devk8s --mem 6G --cpus 2 --disk 30G --cloud-init ./k8s-cloud-init.yml -vvvv`
+3. Log into the VM  
+`multipass shell devk8s`
+4. Create a local cluster using kind. You can find the latest `---image` tags on [Docker Hub](https://hub.docker.com/r/storageos/kind-node/tags)
+`kind create cluster --image storageos/kind-node:v1.22.3 --name portal-dev`
+
+# Creating Credentials foy Your Cluster
+
+1. Open [Ondat Portal](https://portal.ondat.io/dashboard).
+2. Log into your account using your account's credentials.
```
Comment (mvisintin) [inline docs/concepts/portal/installation-portal.md:30]: Or create

File: docs/concepts/portal/installation-portal.md [hunk 12]
```diff
@@ -0,0 +1,50 @@
...
+3. On the lower left-hand side of the screen, open the __Organization__ tab. 
+4. Open the __API Tokens__ tab and select __Create API Token__.
+5. Enter a name for the API token. Note that, if there is another API Token with the same description that token will be replaced by this one.
+6. Copy all information on the page, this will be the only time it will be visible. Make note of the API secret. 
+
+# Installing Ondat on Your Cluster Using the Ondat Portal Manager
```
Comment (mvisintin) [inline docs/concepts/portal/installation-portal.md:36]: Should add a comment about using the values of the previous steps to fill out this command.

File: docs/concepts/portal/applications.md (hunk 6 above)
Comment (mdeneva) [reply docs/concepts/portal/applications.md:6]: Do we even have such thing as not yet deployed apps. If not, I will just remove it.

Comment (mvisintin) [reply docs/concepts/portal/applications.md:6]: Not really, think the whole distinction should be removed

File: docs/concepts/portal/installation-portal.md (hunk 10 above)
Comment (mdeneva) [reply docs/concepts/portal/installation-portal.md:13]: No, actually no idea where it is. If you have it and you can help with that - great. If not have to do some investigation cause I do not have the config copied locally. Thanks

File: docs/concepts/portal/installation-portal.md (hunk 11 above)
Comment (mdeneva) [reply docs/concepts/portal/installation-portal.md:30]: What does that "or create" mean? I do not understand the comment

**Review by mvisintin — APPROVED** [review]

**Comment by cannisondat** [comment]
Please split the PR so the port docs can be a separate PR, the rest can be merged

**Comment by mdeneva** [comment]
Can you review with last changes before the merge of a first release, please? @mvisintin @m-allanson

File: docs/concepts/portal/installation-portal.md (hunk 9 above)
Comment (m-allanson) [reply docs/concepts/portal/installation-portal.md]: I think all the _Local Cluster_ steps should be removed. They're likely to be more confusing than helpful. I have a copy of the `cloud_init.yml` file but it's not up to a standard where we can release it publicly.

These headings and their content should be removed: 
- `# Setting Up a Local Cluster`
- `## Prerequisites`
- `## Procedure`

Maybe there could be a new _Prerequisites_ section, that says something like:

_You will need access to a Kubernetes cluster that meets the [prerequisites for installing Ondat](https://docs.ondat.io/docs/prerequisites/)._

**Comment by cannisondat** [comment]
@m-allanson has given a go ahead - @mdeneva feel free to merge

**Comment by cannisondat** [comment]
@m-allanson could you please confirm if the changes are ok ? If yes, could you please approve the PR?

**Review by m-allanson — APPROVED** [review]

**Pull request merged**